src/System.Text.Json/tests/Serialization/String.ReadTests.cs

```diff
@@ -1,53 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Xunit;
-
-namespace System.Text.Json.Serialization.Tests
-{
-    public static partial class StringTests
```

Comment:
Are we removing any test coverage here or did all of them moved to `ObjectTests`?

Reply:
Just refactoring. All tests still there.